grapple/types/sites.py

```diff
@@ -70,24 +72,21 @@ class Mixin:
             graphene.NonNull(SiteObjectType), enable_search=True, required=True
         )
 
-        # Return all sites.
-        def resolve_sites(self, info, **kwargs):
+        def resolve_sites(self, info, **kwargs) -> QuerySet[Site]:
+            """
+            Return all `Site` objects.
+            """
+
             return resolve_queryset(Site.objects.all(), info, **kwargs)
 
-        # Return a site, identified by ID or hostname.
-        def resolve_site(self, info, **kwargs):
-            id, hostname = kwargs.get("id"), kwargs.get("hostname")
-
-            if id:
-                return Site.objects.filter(pk=id).first()
-            elif hostname:
-                try:
-                    return resolve_site(hostname)
-                except Site.MultipleObjectsReturned as err:
-                    raise GraphQLError(
-                        "Your 'hostname' filter value of '{}' returned multiple sites. Try adding a port number (for example: '{}:80').".format(
-                            hostname, hostname
-                        )
-                    ) from err
+        def resolve_site(self, info, **kwargs) -> Optional[Site]:
+            """
+            Return a single `Site` object, identified by ID or hostname.
```

Comment:
```suggestion
            Return a single `Site` object, identified by ID or hostname.
            If neither ID nor hostname is provided, the function will return None.
```
it will implicitly return None, and there is the type annotation but since there is no explicit `return None` maybe is worth to mention in docstring? 
Or it's too much?

Reply:
Good suggestion - done